### PR TITLE
Added VScode workspace setting

### PR DESCRIPTION
**Pull request by GJSK-Novice**

### CODE REVIEW

File: .vscode/standard_robot.code-workspace [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+	"folders": [
+		{
```
Comment (mac-robomaster) [inline .vscode/standard_robot.code-workspace:3]: QA message 1

**Review by mac-robomaster — CHANGES_REQUESTED** [review]
Left some QA messages, @GJSK-Novice please take a look

File: .vscode/standard_robot.code-workspace (hunk 1 above)
Comment (GJSK-Novice) [reply .vscode/standard_robot.code-workspace:3]: Commit 1817882d7b1899fe13efba1c1d140c116af940f0

**Review by mac-robomaster — APPROVED** [review]
Approved

**Review by mac-robomaster — APPROVED** [review]
Approved

**Pull request merged**